src/stravalib/model.py

```diff
@@ -336,33 +336,8 @@ class SummaryClub(MetaClub, strava_model.SummaryClub):
     dimensions: Optional[list[str]] = None
     localized_sport_type: Optional[str] = None
 
-    @lazy_property
```

Comment:
I can put this back if i'm missing something. i don't see how this is used given members and activities are separate endpoints

e.g.: 

clubs/{id}/activities

Reply:
ahhhh ok @jsamoocha this was a pure miss on my part. i should have looked more closely at the code and what it was doing. i just saw that it was a separate endpoint + the spec and jumped to a conclusion that it didn't belong there. 

that is a really nice feature to have. i've added it back in and it does appear to be working.